service/tbc/tbc_fork_test_data_1.go

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) 2024 Hemi Labs, Inc.
+// Use of this source code is governed by the MIT License,
+// which can be found in the LICENSE file.
+
+package tbc
+
+var tbcForkTestData1 []string = []string{
```

Comment:
This variable appears to be unused.
```
service/tbc/tbc_fork_test_data_1.go:7:5: var tbcForkTestData1 is unused (U1000)
```

Reply:
We will use this once we get to unwinding indexes.